src/openrct2/entity/Guest.cpp

```diff
@@ -4807,7 +4824,7 @@ void Guest::UpdateRideMazePathfinding()
 
     if (IsActionInterruptable())
     {
-        if (Energy > 64 && (scenario_rand() & 0xFFFF) <= 2427)
+        if (Energy > 64 && (scenario_rand() & 0xFFFF) <= 2427 && !climate_is_raining())
```

Comment:
Why this change? Is it just that jumping in the rain probably isn't realistic?

Reply:
Yes, I noticed it looked a bit weird when the guests with umbrellas suddenly jumped (during which they don't hold them of course).

(This was already present before this change - they could have entered the maze before it started to rain)